mlocal/frags/Makefile.stub

```diff
@@ -67,6 +72,15 @@ $(singularity_INSTALL): $(singularity)
 	$(V)install -d $(@D)
 	$(V)install -m 0755 $(singularity) $(singularity_INSTALL) # set cp to install
 
+# runsy
+$(runsy): $(go_OBJ) $(singularity_OBJ)
+	@echo " GO" $@
+	$(V)go build -o $(BUILDDIR)/runsy $(SOURCEDIR)/src/cmd/runsy/cli.go
+$(runsy_INSTALL): $(runsy)
+	@echo " INSTALL" $@
+	$(V)install -d $(@D)
+	$(V)install -m 0755 $(runsy) $(runsy_INSTALL) # set cp to install
```

Comment:
Should remove `# set cp to install` comment. Also while you're at it, can you remove that same comment from line 73?